satpy/tests/reader_tests/test_ami_l1b.py

```diff
@@ -308,3 +321,34 @@ def test_ir_calibrate(self):
         np.testing.assert_allclose(res.data.compute(), expected, equal_nan=True, atol=0.04)
         # make sure the attributes from the file are in the data array
         self.assertEqual(res.attrs['standard_name'], 'toa_brightness_temperature')
+
+        # test GSICS coefficients
```

Comment:
Multiple smaller tests are better that one big test function. Consider splitting this.

Reply:
Done.